python/vyos/ifconfig/ethernet.py

```diff
@@ -258,6 +259,22 @@ def set_rps(self, state):
         # send bitmask representation as hex string without leading '0x'
         return self.set_interface('rps', rps_cpus)
 
+    def set_rfs(self, state):
+        rfs_flow = 0
+        global_rfs_flow = 0
+        ifname = self.config['ifname']
+        queues = glob(f'/sys/class/net/{ifname}/queues/rx-*')
+        if state:
+            global_rfs_flow = 32768
+            rfs_flow = global_rfs_flow/len(queues)
+
+        self._cmd(f'echo {int(global_rfs_flow)} > /proc/sys/net/core/rps_sock_flow_entries')
+
+        for i in range(0,len(queues)):
+            self._cmd(f'echo {int(rfs_flow)} > /sys/class/net/{ifname}/queues/rx-{i}/rps_flow_cnt')
```

Comment:
As written in my previous comment, please use an implementation based on _sysfs_set like we do for rps. You just call things directly making it hatd to debug - please use the interfaces available!

Reply:
Sorry, if I want to set the entire queue of a network card, I may not be able to write to _sysfs_set unless _sysfs_set can accept two parameters.

Reply:
Then please use `_write_sysfs()` and `_read_sysfs()` at least so we get the debug output.

Reply:
Well, I have never used these two functions, can you tell me the method used or provide examples?

Reply:
https://github.com/vyos/vyos-1x/blob/current/python/vyos/ifconfig/control.py#L115-L133

Reply:
done